django_project/changes/views/changelog_github.py

```diff
@@ -260,13 +260,18 @@ def download_all_referenced_images(request, **kwargs):
                             folder_path, '{}'.format(filename))
                         found = os.path.exists(file_path)
                         if found:
+                            # file_path
+                            # e.g /home/web/media/images/entries/img.png
                             file_path_original = file_path
                             n = 0
                             while found:
                                 n += 1
                                 img_name = file_path_original.rsplit('.', 1)[0]
                                 extension = \
                                     file_path_original.rsplit('.', 1)[-1]
```

Comment:
Do not split twice, but rather:
This works because the split only returns two-tuple, so it's easier to read.

```suggestion
                                img_name, extension = file_path_original.rsplit('.', 1)
```

Reply:
But actually we should rather use `os.path.splitext` for this kind of things.